core/shared/src/main/scala/fs2/Pull.scala

```diff
@@ -147,6 +124,28 @@ sealed abstract class Pull[+F[_], +O, +R] {
 
 object Pull extends PullLowPriority {
 
+  implicit class StreamPullOps[F[_], O](private val pull: Pull[F, O, Unit]) extends AnyVal {
```

Comment:
I looked on other random implicit classes in fs2, some of them have `final`, others don't.

Shouldn't final be added here?

Reply:
changed.